.github/workflows/tests.yaml

```diff
@@ -569,20 +569,19 @@ jobs:
 
       - name: Get version
         id: version
-        run: echo "::set-output name=version::v0.0.0-ci.${GITHUB_SHA}-$(date +%Y%m%d%H%M%S)"
```

Comment:
@ogzhanolguncu we need this actually
otherwise the daily check will fail to publish, cause the version exists